hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -211,9 +211,6 @@
                 '''
             }
           ]
-          tags: [// Changing modes randomly can result in unknown values being
-                 // passed between spi_dev/host due to passthrough
-                 "excl:CsrNonInitTests:CsrExclWrite"]
```

Comment:
@eunchan switching mode is fine in block-level. Will this be an issue in chip-level?